Makefile

```diff
@@ -111,7 +126,7 @@ docs/%.8: docs/%.8.md check-gopath
 
 docs: $(MANPAGES)
 
-install: check-gopath
+install: ocid ocic kpod conmon/conmon pause/pause
```

Comment:
funny, because it does require them ... ;-) but it is more silly to require the check-gopath, even tho nothing needs it